package.json

```diff
@@ -59,7 +59,7 @@
     "webpack-cli": "^4.10.0"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "^4.5.4"
+    "@eppo/js-client-sdk-common": "^4.6.1"
```

Comment:
Depends on this to be published https://github.com/Eppo-exp/js-sdk-common/pull/163